### PR TITLE
Spectra v2.2.0

### PR DESCRIPTION
**Pull request by RajkiranBagal**
* New: Introducing the Modal block that will help you to show your own customized popups on the page and turn your visitors into subscribers and leads!
* Improvement: Global - Added hover color accessibility to Spectra block with a button.
* Improvement: Counter - Bar Layout - Added toggle to flip/swap the positions of bar and title.
* Fix: Container - Resolved the issue where the list items overflow inside a Container on responsive devices.
* Fix: Image - Resolved the issue where the alignments are ineffective inside a Container on the front end.
* Fix: Buttons - Page scroll to the top when the hash link is added and refresh on click when the hash is removed.
* Fix: Post Grid - Equal Height is not functional in Post Grid block.
* Fix: Tabs - Removed the WP default Link option from the toolbar for the tabs title field.
* Fix: Testimonial - Block break when we reduce the number of testimonials to 2.
* Fix: Block Transform - Theme pallet color for default theme does not carry forward when block transform.
* Fix: Typography - Resolved Google font console errors.

### CODE REVIEW

**Review by imrantushar — APPROVED** [review]

File: classes/class-uagb-block-js.php [hunk 1]
```diff
@@ -683,5 +683,25 @@ public static function blocks_image_gallery_gfont( $attr ) {
 			UAGB_Helper::blocks_google_font( $caption_load_google_font, $caption_font_family, $caption_font_weight );
 			UAGB_Helper::blocks_google_font( $load_more_load_google_font, $load_more_font_family, $load_more_font_weight );
 		}
+
+		/**
+		 * Adds Google fonts for Modal block.
+		 *
+		 * @since x.x.x
```
Comment (aapatre) [inline classes/class-uagb-block-js.php]: `@since 2.2.0`

File: includes/blocks/modal/attributes.php [hunk 2]
```diff
@@ -0,0 +1,204 @@
+<?php
+/**
+ * Block Information & Attributes File.
+ *
+ * @since x.x.x
```
Comment (aapatre) [inline includes/blocks/modal/attributes.php]: `@since 2.2.0`

File: includes/blocks/modal/block.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Block Information & Attributes File.
+ *
+ * @since x.x.x
```
Comment (aapatre) [inline includes/blocks/modal/block.php]: `@since 2.2.0`

File: includes/blocks/modal/frontend.js.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Frontend JS File.
+ *
+ * @since x.x.x
```
Comment (aapatre) [inline includes/blocks/modal/frontend.js.php]: `@since 2.2.0`

File: includes/blocks/modal/frontend.css.php [hunk 5]
```diff
@@ -0,0 +1,394 @@
+<?php
+/**
+ * Frontend CSS loading File.
+ *
+ * @since x.x.x
```
Comment (aapatre) [inline includes/blocks/modal/frontend.css.php]: `@since 2.2.0`

**Review by AbhijitNage123 — APPROVED** [review]

**Review by aapatre — APPROVED** [review]

**Pull request merged**